src/fuzzer_command_generator.h

```diff
@@ -0,0 +1,28 @@
+#ifndef FUZZER_COMMAND_GENERATOR_H
+#define FUZZER_COMMAND_GENERATOR_H
+
+#include <valkey/valkey.h>
+#include "sds.h"
+
+typedef struct Command {
+    sds *argv;
+    int argc;
+    int size;
+} Command;
```

Comment:
Shall we prefix the various struct around these with Fuzzer* ?

Reply:
Since they're only used within the fuzzer-specific files, I don't think it is required.

Reply:
If they are in the header, they should have a prefix.